lib/component.js

```diff
@@ -113,22 +105,25 @@ function humanName(name) {
   return words.join(' ');
 }
 
-function createComponent(
-  componentName,
-  componentTitle,
-  location,
-  documentation
-) {
+async function createComponent(componentName, componentTitle, location) {
   if (fs.existsSync(location)) {
     console.log('Component directory already exists');
   } else {
     mkdirp(location)
-      .then(() => {
-        const filesArray = ['scss', 'twig', 'yml', 'md'];
-        filesArray.forEach(function(file) {
-          makeComponentFile(componentName, componentTitle, location, file);
-        });
-        console.log(`${componentTitle} created`);
+      .then(async () => {
+        const filesArray = ['scss', 'twig', 'yml', 'md'].map(
+          async ext =>
```

Comment:
The promise returned by `makeComponentFile` isn't being used by this arrow function, so it's probably simple to remove the `async`/`await` here (same reason as mentioned in an earlier comment - the extra `await` forces an unnecessary yield).